core/src/main/scala/org/apache/spark/sql/backwardcomp/logicalPlans.scala

```diff
@@ -0,0 +1,257 @@
+/*
+ * Copyright (c) 2016 SnappyData, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License. See accompanying
+ * LICENSE file.
+ */
+package org.apache.spark.sql.backwardcomp
+
+import java.util.Date
+
+import scala.collection.mutable.ArrayBuffer
+
+import org.apache.spark.sql.catalyst.catalog.{CatalogColumn, CatalogTable, CatalogTablePartition, CatalogTableType, SessionCatalog}
+import org.apache.spark.sql.{AnalysisException, Row, SparkSession}
+import org.apache.spark.sql.catalyst.TableIdentifier
+import org.apache.spark.sql.catalyst.expressions.{Attribute, AttributeReference}
+import org.apache.spark.sql.catalyst.plans.logical
+import org.apache.spark.sql.catalyst.plans.logical.LeafNode
+import org.apache.spark.sql.execution.command.{CreateDataSourceTableUtils, DDLUtils}
+import org.apache.spark.sql.types.{MetadataBuilder, StringType, StructType}
+
+/**
+ * **Copy of RunnableCommand to ensure a compatibility between
+ * Spark 2.0.0 and Snappy Spark 2.0.2**
+ *
+ * A logical command that is executed for its side-effects.  `SnappyRunnableCommand`s are
+ * wrapped in `SnappyExecutedCommandExec` during execution.
+ */
+trait SnappyRunnableCommand extends LeafNode with logical.Command  {
+  override def output: Seq[Attribute] = Seq.empty
+  def run(sparkSession: SparkSession): Seq[Row]
+}
+
+/**
+ * **Copy of DescribeTableCommand to ensure a compatibility between
+ * Spark 2.0.0 and Snappy Spark 2.0.2
+ *
+ * Command that looks like
+ * {{{
+ *   DESCRIBE [EXTENDED|FORMATTED] table_name partitionSpec?;
+ * }}}
+ */
+case class SnappyDescribeTableCommand(
+    table: TableIdentifier,
```

Comment:
Can we avoid "Snappy" prefix for each of the new plan types? Say:

SnappyRunnableCommand => ExecuteCommand
SnappyDescribeTableCommand => DescribeTable